pages/common/neo4j-admin.md

```diff
@@ -0,0 +1,32 @@
+# neo4j-admin
+
+> Manage and administer a Neo4j DBMS (Database Management System).
+> More information: <https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/>.
+
+- Start the DBMS:
+
+`neo4j-admin server start`
+
+- Stop the DBMS:
+
+`neo4j-admin server stop`
+
+- Set the initial password of the default `neo4j` user (prerequisite for the first start of the DBMS):
+
+`neo4j-admin dbms set-initial-password {{database_name}}`
+
+- Create an archive (dump) of an offline database to a file named `database_name.dump`:
+
+`neo4j-admin database dump --to-path={{/path/to/folder}} {{database_name}}`
+
+- Load a database from an archive named `database_name.dump`:
+
+`neo4j-admin database load --from-path={{/path/to/folder}} {{database_name}} --overwrite-destination=true`
```

Comment:
```suggestion
`neo4j-admin database dump --to-path={{path/to/directory}} {{database_name}}`

- Load a database from an archive named `database_name.dump`:

`neo4j-admin database load --from-path={{path/to/directory}} {{database_name}} --overwrite-destination=true`
```


I think this conflicts with another review but we should use the word directory to keep in line with other pages.

Also I am unfamiliar with this command, can we omit the / here

Reply:
I agree with you that this should be a `directory` instead of a `folder` that's what I meant in the previous suggestion. I never used this tool but in the docs I have seen, all the paths were prefixed with `/`, so this is why I suggested adding this.